src/osal.h

```diff
@@ -127,6 +127,8 @@ typedef struct os_ethaddr
    uint8_t addr[6];
 } os_ethaddr_t;
 
+typedef struct pnet pnet_t;
```

Comment:
@hefloryd Please check that this is OK.

Reply:
I have tried to document what the SNMP (sub) agent needs here: https://github.com/rtlabs-com/p-net/blob/master/doc/network_topology_detection.rst#supported-snmp-variables-for-profinet
It is not yet clear to me exactly which parts of `pnet_t` that will be affected. We also consider changing `pnet_t` a little to accommodate multiple ports.

Reply:
The SNMP server is using LLDP-variables as well as some other variabels, such as the location of the device.

Reply:
Added a comment.